tests/unit/position/position_core.js

```diff
@@ -763,4 +761,6 @@ test( "bug #8710: flip if flipped position fits more", function() {
 	}, "no flip - top fits less" );
 });
 
+TestHelpers.testJshint( "position" );
```

Comment:
I still wonder if this could be fixed by `positon: absolute` on the support test element.